plugins/base/src/main/resources/edu/wpi/first/shuffleboard/plugin/base/widget/RelayWidget.fxml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ToggleButton?>
+<?import javafx.scene.control.ToggleGroup?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<StackPane fx:id="root" stylesheets="@relay-widget.css" xmlns="http://javafx.com/javafx/8.0.121"
+           xmlns:fx="http://javafx.com/fxml/1"
+           fx:controller="edu.wpi.first.shuffleboard.plugin.base.widget.RelayWidget">
```

Comment:
No minimum size specified. Causes issues at low tile sizes. 128x64 seems like it could be OK

![relay-no-min-size](https://user-images.githubusercontent.com/6320992/32137149-1f4b8802-bbe8-11e7-9165-030049a3d3fa.PNG)